src/renderer/components/AircraftSection/index.tsx

```diff
@@ -470,6 +472,16 @@ const index: React.FC<TransferredProps> = (props: AircraftSectionProps) => {
         return state.liveries.map((entry) => entry.livery);
     });
 
+    function startMSFS() {
+        const child = require('child_process').execFile;
+        const file = "FlightSimulator.exe";
+        const MSFSisRunningButton = document.getElementById('msfsrunning');
+        MSFSisRunningButton.removeAttribute("hidden");
+        document.getElementById("startMSFS").style.display = "none";
```

Comment:
**issue:** Do not use imperative DOM APIs in a React app.

**suggestion:** Change this to use iodiomatic React constructs.

Reply:
To sum up what Holland said: you shouldn’t be using document selectors as your go-to when using React. Instead of manually getting the element and changing its styling, you should be changing a state that you can use to style the component instead.